src/private-key.ts

```diff
@@ -22,7 +25,21 @@ function findPrivateKey (filepath: string): Buffer | string {
     return fs.readFileSync(filepath)
   }
   if (process.env.PRIVATE_KEY) {
-    return process.env.PRIVATE_KEY.replace(/\\n/g, '\n')
+    let cert = process.env.PRIVATE_KEY
+
+    if (isBase64(cert)) {
+      // Decode base64-encoded certificate
+      cert = Buffer.from(cert, 'base64').toString()
+    }
```

Comment:
👍Good call. I think I mentally avoided this since the content of the RSA key is technically base64 as well, but given the implementation I like this better